content/authorization/guides/system-vendor/system-user/systemregistration/_index.nb.md

```
@@ -14,9 +14,11 @@ Før du kan registrere et system, må følgende være på plass:
   * Tilgang til systemregisteret og systembruker-scopes er innhentet fra Digdir. Prosedyren for dette er beskrevet på [Samarbeidsportalen](https://samarbeid.digdir.no/altinn/kom-i-gang/2868).
   * For testing anbefales det at SBS-organisasjonsnummeret er lagt inn i testmiljøet TT02.
   * For å få opprettet et ekte organisasjonsnummer i TT02, må SBS kontakte servicedesk@altinn.no.
+  * Et gyldig Maskinporten-token eller Altinn-token for å autentisere API-kallet mot systemregisteret.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)